cost/azure/rightsize_managed_disks/azure_rightsize_managed_disks.pt

```diff
@@ -1060,42 +1060,52 @@ script "js_disk_rightsizing_recommendations", type: "javascript" do
     return priceData
   }
 
+  // When looking at the findPremSsdV2Price function check: https://learn.microsoft.com/es-es/azure/virtual-machines/disks-types#premium-ssd-v2-performance
   function findPremSsdV2Price(diskRegion, currentUsedGB, currentUsedIops, currentUsedThroughput) {
     var premSsdV2Pricing = ds_azure_md_pricing[diskRegion]
     if (premSsdV2Pricing.PREMIUM_LRS_PROVISIONED_CAPACITY === undefined) { return }
-    if (currentUsedIops <= 3000 && currentUsedThroughput <= 3000) {
+    if (currentUsedIops <= 3000 && currentUsedThroughput <= 125) {
       return {
         pricePerUnit: Math.round( premSsdV2Pricing.PREMIUM_LRS_PROVISIONED_CAPACITY.pricePerUnit * 720 * currentUsedGB ),
         currencyCode: premSsdV2Pricing.PREMIUM_LRS_PROVISIONED_CAPACITY.currencyCode,
         sizeGiB: currentUsedGB,
-        // Premium SSD v2 provides a baseline performance of 3,000 IOPS and 125 MB/s for any disk size
+        // Premium SSD v2 provides a baseline performance of 3000 IOPS and 125 MB/s for any disk size without extra charge.
         IOPS: 3000,
         throughput: 125,
         unitOfMeasure: premSsdV2Pricing.PREMIUM_LRS_PROVISIONED_CAPACITY.unitOfMeasure
       }
     }
-    var diskIopsAndThroughputDontFitInDiskSize = true
     var suggestedDiskSize = currentUsedGB
     var maxIops = 0
     var maxThroughput = 0
-    while (diskIopsAndThroughputDontFitInDiskSize) {
-      maxIops = suggestedDiskSize * 500
-      maxThroughput = maxIops * 0.25
+    while (true) {
```

Comment:
This is a bad coding pattern. You should refactor this to not rely on while (true)

Instead, you should replace true with whatever condition breaks the loop. If you need this to occur after the loop iterates, you can use do while() to have the check occur after the loop.

Reply:
Wow I did not know `do while` existed in Javascript, Implementing this change...

Reply:
I'm done finishing the changes and also verified them:

![image](https://github.com/user-attachments/assets/85acfb39-9b0b-4278-8f82-8069bef60010)